mysql-test/suite/tokudb/t/fast_update_blobs_fixed_varchar.py

```diff
@@ -9,19 +9,24 @@ def main():
     print "source include/have_tokudb.inc;"
```

Comment:
Likewise

Reply:
Sorry, I did not understand, "likewise" what? Could you please clarify?

Reply:
This points to "The generated testfiles are still too large (as diff says +80K added lines) ... "

Reply:
Ok, I got it. The comment is not about the certain line, but about the whole file.

Reply:
See new version of ex-'python generated' tests,